test/fixtures/non-string-argument/expected.js

```diff
@@ -0,0 +1,21 @@
+new Promise(resolve => {
+  resolve(require(new String({ 'answer': 42 })));
```

Comment:
these should absolutely not use `new String` - that produces a string object, whereas `ToString` produces a primitive. It should simply use `String()`.

However, it's also worth noting that `require` in node already coerces it to a string, so I'm not sure it's needed.